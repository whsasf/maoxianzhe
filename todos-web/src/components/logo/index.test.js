import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Logo from "./index";

it("should renders correctly", () => {
  const tree = renderer.create(<Logo>somthing</Logo>).toJSON();
  expect(tree).toMatchSnapshot();
});
