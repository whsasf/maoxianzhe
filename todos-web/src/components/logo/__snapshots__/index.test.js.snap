// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders correctly 1`] = `
.c0 {
  -webkit-animation: fvtopB infinite 20s linear;
  animation: fvtopB infinite 20s linear;
  width: 60px;
  height: 31px;
  margin: 16px 24px 16px -16px;
  float: left;
}

<img
  className="c0"
  src="logo.svg"
>
  somthing
</img>
`;
